samples/buildSrc-test-junit4/README.md

```diff
@@ -0,0 +1,7 @@
+buildSrc-test-junit4
+====================
+
+Sample project setup to show how you can test your _buildSrc_ custom build logic using
+Junit4 Unit tests.
```

Comment:
Junit5 is already released.
I would recommend to use an exmaple with Junit5...

Reply:
we still use JUnit4, this is why the sample is based on it.
I'd be happy to provide more samples e.g. using JUnit5 or Spek if there's interest.

Reply:
@StefMa 
is it acceptable for you to keep the junit4 example or would you prefer to have a junit5 one?

Reply:
I am not a maintainer of this Repo nor I work at gradle 🙃. I, personally, would show an example with junit 5 because it's the latest version. There is literally no need to stick at junit 4.

Reply:
Ok thanks.

 I have created a new PR (https://github.com/gradle/kotlin-dsl/pull/1317) and will close this one.